test/clustertest/testplugin/TestPlugin.cpp

```diff
@@ -104,7 +104,14 @@ bool BedrockPlugin_TestPlugin::peekCommand(SQLite& db, BedrockCommand& command)
         request["Host"] = "www.google.com";
         auto transaction = httpsManager.httpsDontSend("https://www.google.com/", request);
         command.httpsRequests.push_back(transaction);
-        thread([transaction, request](){sleep(35);transaction->s->send(request.serialize());}).detach();
+        if (command.request["neversend"].empty()) {
```

Comment:
What is this `neversend` param?

Reply:
The existing behavior was to wait 35 seconds and then send this HTTP request, making us get the response 35 seconds later. This would expect that we *would* eventually send the command, and then look for a response, which would access the command that sent it in the first place, because the sending was done on a detached thread.

I added `neversend` since we don't actually need this to send at all, so long as it doesn't have a response when our timeout is hit. I didn't want the handler for the response to look up a deleted command (because it had returned 555) and crash.